### PR TITLE
Refactor - streaming AirQo BAM data

### PR DESCRIPTION
**Pull request by NoahNsimbe**
**_WHAT DOES THIS PR DO?_**

- [x] Enables support for storing unclean and clean BAM measurements
- [x] Deletes unused methods.

**_WHAT ISSUES ARE RELATED TO THIS PR?_**

- Jira cards
    - [PLAT-1189](https://airqoteam.atlassian.net/browse/PLAT-1189)
- GitHub issues
    - Closes #<enter GitHub issue number here>

**_HOW DO I TEST OUT THIS PR?_**

**_WHICH ENDPOINTS SHOULD BE READY FOR TESTING?:_** 

**_IS THERE ANY JENKINS CONSOLE LINK FOR CODE COVERAGE AND BUILD INFO?_**

**_ARE THERE ANY RELATED PRs?_**

### CODE REVIEW

File: src/airflow/airqo_etl_utils/config.py [hunk 1]
```diff
@@ -97,5 +93,39 @@ class Config:
...
+        2: "altitude",
+        3: "wind_speed",
+        4: "satellites",
+        5: "hdop",
+        6: "device_temperature",
+        7: "device_humidity",
+        8: "temperature",
+        9: "humidity",
+        10: "pressure",
+    }
```
Comment (NoahNsimbe) [inline src/airflow/airqo_etl_utils/config.py:128]: Configurations for BAM  devices. @Baalmart

Comment (Baalmart) [reply src/airflow/airqo_etl_utils/config.py:128]: Thanks @NoahNsimbe , this LGTM. Thanks.

**Review by Baalmart — COMMENTED** [review]
For the purpose of the bus factor and the phased onboarding of teammates, you might want to add some more guiding notes at some more key sections of your changes. @NoahNsimbe 

One way to do it painlessly is to just add a review comments for each changed file. That way, we shall steadily improve the quality of our reviews.

Thanks!

File: src/airflow/airqo_etl_utils/schema/bam_raw_measurements.json [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+    "type": "FLOAT",
+    "mode": "NULLABLE",
+    "description": "%. Filter Relative Humidity"
+  },
+    {
+    "name": "status",
+    "type": "INTEGER",
+    "mode": "NULLABLE"
+  }
+]
```
Comment (NoahNsimbe) [inline src/airflow/airqo_etl_utils/schema/bam_raw_measurements.json:107]: BigQuery schema @Baalmart

File: src/airflow/airqo_etl_utils/airqo_api.py [hunk 3]
```diff
@@ -91,95 +91,44 @@ def calibrate_data(self, time: str, data: pd.DataFrame) -> list:
             print(ex)
             return []
 
-    def get_calibrated_values(self, time: str, calibrate_body: list) -> list:
```
Comment (NoahNsimbe) [inline src/airflow/airqo_etl_utils/airqo_api.py:94]: Deleting unused methods. @Baalmart

**Pull request merged**